src/main/java/frc/robot/subsystems/Carousel.java

```diff
@@ -12,29 +12,59 @@
 import com.revrobotics.ColorSensorV3;
 import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.Robot;
 
 public class Carousel extends SubsystemBase {
     WPI_TalonSRX spinner;
     Encoder carouselEncoder;
     ColorSensorV3 colorSensor = new ColorSensorV3(I2C.Port.kMXP);
     int ballCount = 0;
-
+    // TODO: remove when removing old carousel command
+    public double currentCheckpoint = 0;
+    public double lastCheckpoint = 0;
```

Comment:
I think these can be removed now?

Reply:
ah, if forgot about those, I just pushed that